pcsx2/DebugTools/DebugInterface.cpp

```diff
@@ -700,7 +700,8 @@ bool R5900DebugInterface::isValidAddress(u32 addr)
 		case 0xA:
 		case 0xB:
 			// [ 8000_0000 - BFFF_FFFF ] kernel
-			return true;
+			if(lopart >= 0xFC00000)
```

Comment:
Space after if.

Reply:
I'll fix in a new commit if it gets squashed later, because that particular change is in a commit from @F0bes